rdt-open-reader/src/covidIndonesia/resources/sample-collection-form_in.properties

```diff
@@ -20,12 +20,12 @@ sample-collection-form-original.step2.health_facility_type.v_required.err = Bagi
 sample-collection-form-original.step5.qr_code_reader.v_numeric.err =
 sample-collection-form-original.step1.next_label = TERUS
 sample-collection-form-original.step4.title = Sampel
-sample-collection-form-original.step2.name_of_primary_care_facility.options.other.text = Lainnya
+sample-collection-form-original.step2.name_of_health_facility.options.other.text = Lainnya
```

Comment:
@vincent-karuri I noticed these are properties for a different step in the `sample-collection-form_in.properties` file. 
I was thinking maybe I shouldn't change these?

Reply:
The property file should be generated using `JMAG` acting on the `*-original.json`. The keys changed a bit after a form refactor, but these should reflect as new keys in Transifex - so not a problem.

Reply:
`JMAG` should also be used to generate the `*.json` files from the `*-original.json` files.

Reply:
So the flow is -> modify the `*-original.json`, generate 2 asset files against it using `JMAG` -> commit all three files. 

A caveat here is that for the placeholder-injected form generated by `JMAG`, remove the `-original` suffix in the `properties_file_name` field before commiting.

Reply:
Cool. Thanks. Got the JMAG to generate the asset files after registering the Covid flavor widgets